corehq/apps/reports/filters/users.py

```diff
@@ -340,6 +340,10 @@ def filter_context(self):
     @classmethod
     def user_es_query(cls, domain, mobile_user_and_group_slugs, request_user):
         # The queryset returned by this method is location-safe
+        q = user_es.UserES().domain(domain)
+        if not any(mobile_user_and_group_slugs):
+            return q.filter(filters.match_all())
```

Comment:
You can just return `q` here - `.filter(filters.match_all())` is a no-op.

Reply:
https://github.com/dimagi/commcare-hq/pull/21923/commits/143be5d91b396d20fdc465d32d7fd6806f8a9f35